<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageFive</class>
 <widget class="QWidget" name="PageFive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/bg_page_5.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emergencyTransmitter">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>317</y>
     <width>422</width>
     <height>364</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/emergency_transmitter.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickableLabel" name="smallEmergencyTransmitter">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>536</y>
     <width>100</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/small_emergency_transmitter.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sosButton">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>530</y>
     <width>144</width>
     <height>113</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/bg_sos_button);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MovableLabel" name="treeTrunk1">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>538</y>
     <width>372</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/tree_trunk_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MovableLabel" name="treeTrunk2">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>573</y>
     <width>372</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/tree_trunk_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>smallEmergencyTransmitter</zorder>
  <zorder>treeTrunk1</zorder>
  <zorder>treeTrunk2</zorder>
  <zorder>emergencyTransmitter</zorder>
  <zorder>sosButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>Model/clickablelabel.h</header>
  </customwidget>
  <customwidget>
   <class>MovableLabel</class>
   <extends>QLabel</extends>
   <header location="global">Model/movablelabel.h</header>
   <slots>
    <signal>labelMove()</signal>
    <signal>mouseRelease()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
